### ISSUE TITLE
i have several smurfs that i am certain use cheats to snipe me.

### ISSUE THREAD

**Issue by bradleydragon** [ValveSoftware/Dota2-Gameplay#12418]
### Description

there are 3-4 smurfs that end up in my games. they go 30+ kills because they use maphacks vs me to kill me the moment i leave base or even IN the base. the games i will post are all within 3 days.

### Screenshots

![Screenshot 2023-10-01 212748](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/c7dbfb75-4111-406a-b1d4-0b77a74e0059)
![Screenshot 2023-10-01 212644](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/92d25bc6-165c-42bf-a17c-3cb6903ca160)
![Screenshot 2023-10-01 212529](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/fb17f98c-0984-43cb-9b4e-17833a57f641)
![Screenshot 2023-10-01 212402](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/01e37dab-70de-45a9-a1e1-1063e4661acd)
![Screenshot 2023-10-01 212318](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/6b1face7-9638-4cf9-b66b-ce2b86071123)
![Screenshot 2023-10-01 212229](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/154e4603-136a-4596-89b7-8af3857087e8)
![Screenshot 2023-10-01 212036](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/646c4c90-3501-4722-9593-2c760631dd0d)
![Screenshot 2023-10-01 211927](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/f6657a76-f158-43f9-b333-b7db0d5dfb4d)
![Screenshot 2023-10-01 211835](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/b4901b06-1633-4ceb-bfad-c9ccc7bd62ef)
![Screenshot 2023-10-01 211744](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/632f8485-3ee9-4225-bc3a-b2016e5eb1db)
![Screenshot 2023-10-01 212903](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/20a560d1-cf78-4afd-a7ba-0e451ae06bf7)
![Screenshot 2023-10-01 212849](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/34447621-4da3-46d6-b8cd-1676105da785)
![Screenshot 2023-10-01 212830](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/182ac5a4-87e3-4b57-88f3-af349e2d6556)

**Comment by yoyoinneverland** [comment]
Your MMR is so low you're playing with the same people around the same MMR...

**Comment by TobyFlendersonFromHR** [comment]
they are smurfing sure but prob not hacking, you just suck.

**Comment by bradleydragon** [comment]
no they use cheats. they constantly find me in fow without vision.  had a 53 death batrider game where they would jump me IN FOUNTAIN to kill me the moment i was targetable.

**Comment by bradleydragon** [comment]
> Your MMR is so low you're playing with the same people around the same MMR...

explain exactly the likelyhood of being vs the exact same 3 smurfs back to back. with little to no other repeats of plays on either team.

**Comment by bradleydragon** [comment]
here. this is a game with 2 of the newer smurf accounts. pa goes after me constantly in fow without vision of me. and early on when i dc the slark ( who fed rapiers all game) makes me break my items. 7356050184 i dont even need to say the smurfs names. the scores alone are proof, we only won because the other team didnt want the smurf to win.

**Comment by TobyFlendersonFromHR** [comment]
Yes, i will personally ban them from my matchmaking by placing them on my avoid list if you can prove it. But i doubt they are cheating.

> no they use cheats. they constantly find me in fow without vision. had a 53 death batrider game where they would jump me IN FOUNTAIN to kill me the moment i was targetable.

Show match ID and time for this

> when i dc the slark ( who fed rapiers all game) makes me break my items

This can never happen, they can't break or drop your items even if you are disconnected.

I think you are making up these stories. Having 8000 matches and being 135 mmr is just a joke made up by a bullshit artist.

**Comment by bradleydragon** [comment]
I literally just showed the match id in my post. learn to read. it
happens early game when all i have is boots and a soul ring. after i
channel my e i dc. and if you watch from slark's perspective he is
controlling my hero and making me break my items.

**Comment by bradleydragon** [comment]
even so the smurf was directly trolling me. watch more of the game and you
will see pa with no vision of me make a beeline to kill me multiple times.

**Comment by TobyFlendersonFromHR** [comment]
No sir, you already proved that you are a liar. I have decided to NOT ban any of the smurfs in your matches. You may request another hearing after playing another 8000 matches, hopefully by then you will have added a few IQ points and no longer have your persecutory delusion.

Have a good day!

**Comment by bradleydragon** [comment]
no one asked you and im not lying. maybe look at the proof.

**Comment by yoyoinneverland** [comment]
> > Your MMR is so low you're playing with the same people around the same MMR...
> 
> explain exactly the likelyhood of being vs the exact same 3 smurfs back to back. with little to no other repeats of plays on either team.
Low MMR, smaller pool of people. If you wish to not see them again, it's best you team up with friends until you're out of MMR hell.

**Comment by bradleydragon** [comment]
are you actually stupid? i just showed you that i have been vs teh same
people repeatedly in the past 3 days. they have been hounding me for MONTHS
hell A YEAR. im sick and tired of these cheating pos. i want them banned
and gone. i literally report them every chance i get. nothing happends.

**Comment by bradleydragon** [comment]
https://cdn.discordapp.com/attachments/872924113725956106/1157088746647334973/image.png?ex=651dedde&is=651c9c5e&hm=3b18433c7bc149d2753a151c4f0d58379559f67f7f7b7ff220209c824e404f63&
https://www.opendota.com/players/920760300
https://www.opendota.com/players/302403881
both used russian bots to get down to my mmr and to quickly lose mmr they
gain from beating me.

**Comment by bradleydragon** [comment]
https://www.dotabuff.com/players/302403881

**Comment by bradleydragon** [comment]
https://www.opendota.com/matches/7354654461

**Comment by bradleydragon** [comment]
need MORE proof they use cheats?

**Comment by yoyoinneverland** [comment]
Yup, this is what I get. Good job Bradley Dragon. You're a very rude human being.

**Comment by bradleydragon** [comment]
maybe because i have had these cheaters sniping my games for over a year. im tired of it.

**Comment by yoyoinneverland** [comment]
Get recalibrated.

**Comment by bradleydragon** [comment]
i have... i have like 6 months left. for the next one. idk why you cant ban
people obviously using bots to get forced 10 min losses. isnt that against
tos?

**Comment by TobyFlendersonFromHR** [comment]
8000 matches
135 mmr

Either you are intentionally tanking your mmr or you really really really suck.

Also you are delusional to think that smurfs/bots are targeting you, you are just a collateral for them (and it doesn't help that you do indeed suck at this game).

**Comment by bradleydragon** [comment]
........................... are you really this daft? these people use
maphacks. every. single. match. when they are vs me. and when they are on
my team they grief me. idk how much proof i need to show you. did you even
bother looking at the screenshots?

**Comment by bradleydragon** [comment]
![Screenshot 2023-10-07 001004](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/09ce9973-5c24-46fc-9955-03952797051d)
![Screenshot 2023-10-07 001108](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/5067465b-e1df-4821-8c0c-0916edf3c83a)
![Screenshot 2023-10-07 001138](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/d98fbca1-8c04-4ec2-9adf-6ad3ad5b11f3)
![Screenshot 2023-10-07 001210](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/5c952b4f-7192-40d5-9a53-e063721ebdf6)
![Screenshot 2023-10-07 001302](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/401d5f4e-2b60-4c87-8618-85e85edb6345)
this proof enough im being targetted?

**Comment by bradleydragon** [comment]
![Screenshot 2023-10-07 001443](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/1189d843-3561-4c97-af01-1af8ecff48c3)

**Comment by bradleydragon** [comment]
![Screenshot 2023-10-07 001540](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/33ac2f75-4533-40a4-acaa-5271bb07baad)

**Comment by yoyoinneverland** [comment]
Welp, now I see it. Best thing you can do is roll a secondary account and e-mail Valve about this. Possibly Gabe too.

**Comment by TobyFlendersonFromHR** [comment]
>  when they are on my team they grief me. idk how much proof i need to show you

How about you show proof of this one? When are they on your team and grief you? Any recent games of this???

Otherwise all you have shown is that you are getting beaten by smurfs, they aren't targeting you at all, they just perform well in almost every game they play.

**Comment by yoyoinneverland** [comment]
> > when they are on my team they grief me. idk how much proof i need to show you
> 
> How about you show proof of this one? When are they on your team and grief you? Any recent games of this???
> 
> Otherwise all you have shown is that you are getting beaten by smurfs, they aren't targeting you at all, they just perform well in almost every game they play.

I think he showed us their profiles. It seems they are indeed targeting the guy, somehow.

**Comment by TobyFlendersonFromHR** [comment]
Nope!!!

Their profiles don't show they have dotaplus, so it's not like they are putting this guy on the avoid list so they can always match against him.

This guy is just someone who is losing to smurfs and is upset about it, which is fine, except when this guy comes up with a delusional theory that the smurfs are actively hunting him like he has a bounty on his head.

**Comment by bradleydragon** [comment]
i have them on my avoid list. tho i have been letting them slide off it
because it seems slightly more effective to report them for griefing when
they are on my own team. reporting them for smurfing and map cheating has
so far had zero effect.

**Comment by bradleydragon** [comment]
> > > when they are on my team they grief me. idk how much proof i need to show you
> > 
> > 
> > How about you show proof of this one? When are they on your team and grief you? Any recent games of this???
> > Otherwise all you have shown is that you are getting beaten by smurfs, they aren't targeting you at all, they just perform well in almost every game they play.
> 
> I think he showed us their profiles. It seems they are indeed targeting the guy, somehow.

![Screenshot 2023-10-07 024309](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/9f4f198c-f2f7-41a6-ba80-f043bfe5e75c)
fed rapier.
![Screenshot 2023-10-07 024655](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/7a589899-17ee-4cb3-bcf9-b61a66e76eb9)
didnt help all game. just afk farmed "trying to one up me in farming" not trying to win whatsoever.
![Screenshot 2023-10-07 024841](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/e6020a96-6010-4eaf-b029-cc5cfcf48ae0)
literally just a build for kill stealing and griefing. won despite his best efforts to screw with me.

**Comment by bradleydragon** [comment]
![Screenshot 2023-10-07 025107](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27696694/20a4f211-0daa-48ac-9d16-314a7b719318)
this build speaks for itself. kept griefing me as troll. turning me to stone and throwing me at the enemy. fed 2 gems.

**Comment by TobyFlendersonFromHR** [comment]
Feeding rapier, afk farming, kill stealing? This happens in all brackets, it just happens more in herald bracket because everyone there sucks. If they are really griefing you then they would run down mid and die 50 times or afk farm with zero kills or deaths all game, has that ever happened? 

And are these all just one-off cases? Can you show multiple games where the SAME PLAYER has done the same sort of griefing ONLY TO YOU?

No, of course you can't, because it never happens like that.

Ok that earth spirit throwing you into enemy is griefing but that's just one game. And all you need to do is turn on the 'disable help' option. Also his score is 6/3/26 so they still had a positive impact for for your team overall.

Also show your behavior score, if you are low behavior score then these matches will happen more often and it will be your fault for being toxic.

**Comment by bradleydragon** [comment]
9k behavior score. it went up since they cant spam report me to
artificially lower it. also you dont see the same guilds from teh earlier
screenshots? these same people griefing me are teh same people smurfing on
enemy team. how the hell are you so blind?

**Comment by yoyoinneverland** [comment]
Well, Brad, have you e-mailed Valve yet?

**Comment by bradleydragon** [comment]
> Feeding rapier, afk farming, kill stealing? This happens in all brackets, it just happens more in herald bracket because everyone there sucks. If they are really griefing you then they would run down mid and die 50 times or afk farm with zero kills or deaths all game, has that ever happened?
> 
> And are these all just one-off cases? Can you show multiple games where the SAME PLAYER has done the same sort of griefing ONLY TO YOU?
> 
> No, of course you can't, because it never happens like that.
> 
> Ok that earth spirit throwing you into enemy is griefing but that's just one game. And all you need to do is turn on the 'disable help' option. Also his score is 6/3/26 so they still had a positive impact for for your team overall.
> 
> Also show your behavior score, if you are low behavior score then these matches will happen more often and it will be your fault for being toxic.

disable help does not prevent tiny from tossing you if he has a helmdom, also does not prevent lifestealer from hiding insid eyou. grefing yoru exp all game and coming out only to steal kills.

**Comment by TobyFlendersonFromHR** [comment]
> Well, Brad, have you e-mailed Valve yet?

I agree, best thing to do is to email valve about your case, after evaluating it they will send you the contact details of your local psychiatric hospital since you are in desperate need of it.

**Comment by bradleydragon** [comment]
how are you this delusional. did you not look at the profile pictures i
sent of these guys? "brad is a nonce" "brad is a pedo" how are you that
fking blind man. it is obvious they are talking about me. they smear me
every chance they get.

**Comment by Cameron2920** [comment]
> Stream sniping isn't cheating. It is against TWITCH ToS for STREAMERS to stream snipe other STREAMERS but Valve couldn't care less about Twitch ToS.
> 
> ![image](https://user-images.githubusercontent.com/21988932/273454516-9ff82163-8619-44cb-8d25-6eb5954dbebb.png)
> 
> BTW, is this the same "bradley dragon"??? WTF.
> 
> ![image](https://user-images.githubusercontent.com/21988932/273454593-18e35e6e-fa08-4610-8255-86241c4669f7.png)
> 
> ![image](https://user-images.githubusercontent.com/21988932/273454632-c818c33a-7a01-4031-8e13-82375c907e7f.png)
> 
> ![image](https://user-images.githubusercontent.com/21988932/273454835-920aa180-2ff5-483c-86f5-7fa5c4923923.png)
> 
> ![image](https://user-images.githubusercontent.com/21988932/273454709-d3276c85-c8be-487e-92b1-30d89fe5d95d.png)
> 
> @jeffhill @wykrhm

Yes, I do believe it is the same bradleydragon. I'm surprised Valve hasn't banned him from Dota. There should be no room in the game for that kind of repulsive behaviour.

**Comment by yoyoinneverland** [comment]
And now the truth is out... I feel bad for your twitch channel, Brad.

**Comment by JimxDangle** [comment]
How the tables have turned.

**Comment by JimxDangle** [comment]
My boy.... my sweet boy...

**Comment by curious-envelope** [comment]
![unhinged](https://github.com/ValveSoftware/Dota2-Gameplay/assets/147332984/dfb0df1d-d498-4b2e-84a1-563de0a94aed)

whats wrong brad, why would you want to delete this thread? you sure put a lot of bulletproof evidence here that you are the victim and it would be shame to toss all of that into the trashcan, no??

**Comment by yoyoinneverland** [comment]
I wonder if we were being trolled.

**Comment by curious-envelope** [comment]
![degen](https://github.com/ValveSoftware/Dota2-Gameplay/assets/147332984/17263e8c-a0b6-4be7-8098-e2793f318098)

not at all!

**Comment by yoyoinneverland** [comment]
Discord screencaps? Alright guys, who are you and what's going on?

**Comment by JimxDangle** [comment]
[
![IMG_20211109_120058749](https://github.com/ValveSoftware/Dota2-Gameplay/assets/133048485/8c1ac0e3-792c-45e3-bf12-595ecd165a91)
](url)

**Comment by WombaTharrier** [comment]
> Discord screencaps? Alright guys, who are you and what's going on?

Sorry about all the trolls. Bradley is a well known sex pest and dota 2 streamer. He has acquired quite the following of people trying to ruin his life because of him constantly berating and belittling his teammates. He posted this github thread in his discord saying that he was in touch with valve employees. So, now you get to see all of brad's dirty laundry.

**Comment by MrMistofolees** [comment]
> Discord screencaps? Alright guys, who are you and what's going on?

Bradley Dragon is a well known Dota2 Twitch Streamer ( https://www.twitch.tv/bradley_dragon )who has been below 1000 mmr for several years. He is currently below 100 mmr, in part because he is terrible at the game, but also because his viewers streamsnipe him. Bradley is autistic and possibly schizophrenic, most games he loses he is convinced there is an opponent hacking and calls them out - see his lost match chatlogs for evidence of this: https://www.opendota.com/players/80597991/matches

Bradley is seriously mentally unwell and his claims and feedback about the game should be utterly ignored. He has also been implicated in several disgusting scandals as shown in the chat logs above, which are all legitimate. He was involved in sexually molesting his brother at a young age, is obsessed with posting photographs of his genitals online, and has been tied to zoophilia and "toilet" fetish posts online also. The best way to deal with him is to ignore him. He has already been banned on /r/dota2 and from their discord for this exact same thing.

**Comment by Bredleydragon** [comment]
CAN YOU GUYS PLEASE SHUIT THE UFCK UP!!!! STOP TARGETTING ME I AM INNOCENT! THIS IS BLATANT GANG STALKING YOU SEE TEH EVIDENCE

**Comment by curious-envelope** [comment]
> Discord screencaps? Alright guys, who are you and what's going on?

sorry for teh confusion, perhaphs this doc could help clear things out

**Comment by yoyoinneverland** [comment]
> CAN YOU GUYS PLEASE SHUIT THE UFCK UP!!!! STOP TARGETTING ME I AM INNOCENT! THIS IS BLATANT GANG STALKING YOU SEE TEH EVIDENCE

![1695620468981419](https://github.com/ValveSoftware/Dota2-Gameplay/assets/130618708/f443659c-ce5c-46bd-8a61-12c9d11f215c)

**Comment by WombaTharrier** [comment]
![DiscordCanary_W0VrbnQl7e](https://github.com/ValveSoftware/Dota2-Gameplay/assets/75311496/1108dab3-1c61-4715-80ab-2f18785c5369)
This is how bradley feels about your feedback btw

**Comment by Michael-Tobys-Boss** [comment]
Hello Bradley! I am Michael, Toby's boss. I would like to apologize on his behalf for being so unprofessional. If you are ever in need of some paper, just call me at the Scranton branch for some exclusive deals!

**Comment by TobyFlendersonFromHR** [comment]
Holy moly, the garbage has been let out. I knew this guy was mentally impaired, no sane person could have 8000 games and still be 100 mmr. Good to know this guy is getting caught out.

**Comment by Michael-Tobys-Boss** [comment]
Toby, I'm having hard time believing yoh arent one of the viewera but in case you truly arent and are just some random person I think I have to tell you this. Bradley thought he was speaking with someone from Valve's HR department when talking to you. He was going off on about "idiot valve hr employees" a couple days ago before we had found this thread he made

**Comment by TobyFlendersonFromHR** [comment]
Ahaha I didn't know this guy at all. Who even streams herald mmr games? I will say tho, he looks exactly as I expected.

**Comment by Michael-Tobys-Boss** [comment]
https://www.twitch.tv/videos/1944936322?t=03h02m08s

He is quite entertraining tbh. Here's the vod when he speaks about you for couple of minutes. 3h 2min 8s in the VOD if the link doesnt go to the right time stamp until around 3h 6min

**Comment by TobyFlendersonFromHR** [comment]
AHAHAHAHAHAHA THIS GUY THOUGHT I WAS ACTUALLY FROM VALVE'S HR DEPARTMENT...AND THAT SOMEONE FROM HR WAS GOING TO DEAL WITH HIS SMURF ISSUE

God damn, when I saw he was 100 mmr with 8k games, i thought he was just deliberately losing his mmr so he could feel good about beating legit herald players. But then I checked his dotabuff and saw that he was just a legit herald himself. Absolutely baffled by how someone can have so much time in this game, yet still manage to be worse than 99.9% of the ranked playerbase.

This guy was able to overcome a statistical impossibility by being so retarded. I don't know if I should be amazed or disgusted. (Either way, I do honestly feel bad for making fun of a severely retarded person, even if he is toxic as well, he's too retarded to realize that he's being toxic)
